### PR TITLE
README for pbs enabled devnet

### PR DESCRIPTION
**Pull request by jinmel**

### CODE REVIEW

File: pbs/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+$ BUILDER_IMAGE=<YOUR_OP_GETH_BUILDER_IMAGE> DEVNET_BUILDER=true make devnet-up
+```
+
+To additionally enable load testing through [tx-fuzz](https://github.com/MariusVanDerWijden/tx-fuzz), you can run the following command:
+
+```shell
+$ BUILDER_IMAGE=<YOUR_OP_GETH_BUILDER_IMAGE> DEVNET_LOAD_TEST=true DEVNET_BUILDER=true make devnet-up
+```
+
+If the `BUILDER_IMAGE` is not set, the devnet will use the image from `flashbots/op-geth:latest`.
```
Comment (avalonche) [inline pbs/README.md]: this should be at the start of as the user doesn't need to build the docker image anymore to run the devnet

File: pbs/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## How to run the devnet
+
+You first need to build the op-geth docker image with builder API support.
+
+```shell
+$ git clone git@github.com:flashbots/op-geth.git
+$ cd op-geth
+$ docker build . -t <YOUR_OP_GETH_BUILDER_IMAGE>
+```
```
Comment (avalonche) [inline pbs/README.md:27]: I think this can be in a different section for users who want to use their custom op-geth node

File: pbs/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+If the `BUILDER_IMAGE` is not set, the devnet will use the image from `flashbots/op-geth:latest`.
+
+## Configuration
+
+These are the configuration options to enable PBS for the devnet.
+
+### Sequencer
+
+There are three flags that congifure the sequencer to request payloads from builder API endpoint:
```
Comment (avalonche) [inline pbs/README.md]:
```suggestion
There are three flags that configure the sequencer to request payloads from builder API endpoint:
```

File: pbs/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+- `l2.builder.enabled`: Enable the builder API request to get payloads built from the builder.
+- `l2.builder.endpoint`: The URL of the builder API endpoint.
```
Comment (avalonche) [inline pbs/README.md]: This endpoint should follow https://github.com/ethereum-optimism/specs/pull/116/files#diff-3424f00d095fe96d07f55b760552c1555a6550c7417a375a7027cae4e51083a6R88 where the sequencer has a predefined set of builder pub keys and endpoints. 
The pubkey can be embedded in the url like the relay endpoint e.g. `https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net`

File: pbs/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- `eventstream.addr`: The address of the eventstream server.
+- `eventstream.port`: The port of the eventstream server.
+
+
+### builder-op-geth
+
+These are the builder flags to enable the builder service in op-geth:
+
+- `builder`: Enable the builder service.
+- `builder.beacon-endpoints`: The op-node address to get the payload attributes from. Should be set the the builder-op-node.
```
Comment (avalonche) [inline pbs/README.md]:
```suggestion
- `builder.beacon-endpoints`: The op-node address to get the payload attributes from. Should be set to the builder-op-node.
```

File: pbs/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- `builder.block_retry_interval`: The interval to retry building the payload.
+- `builder.block_timeout`: The timeout to build the payload.
```
Comment (avalonche) [inline pbs/README.md]:
```suggestion
- `builder.block_time`: The block time of the network
```

**Review by avalonche — COMMENTED** [review]
It'll also be helpful to note what's the default values of the new flags.

File: pbs/README.md (hunk 4 above)
Comment (jinmel) [reply pbs/README.md]: will address this when we have a builder signature verification.

**Pull request merged**